0.5/index.bs

```diff
@@ -399,25 +395,32 @@ The "labels" group is not itself an image; it contains images. The pixels of the
 [`uint8`, `int8`, `uint16`, `int16`, `uint32`, `int32`, `uint64`, `int64`]. Intermediate groups between "labels" and the images within it are allowed, 
 but these MUST NOT contain metadata. Names of the images in the "labels" group are arbitrary.
 
-The `.zattrs` file associated with the "labels" group MUST contain a JSON object with the key `labels`, whose value is a JSON array of paths to the 
+The OME-Zarr Metadata in the `zarr.json` file associated with the "labels" group MUST contain a JSON object with the key `labels`, whose value is a JSON array of paths to the 
 labeled multiscale image(s). All label images SHOULD be listed within this metadata file. For example:
 
 ```json
 {
-  "labels": [
-    "cell_space_segmentation"
-  ]
+  ...
+  "attributes": {
+    "ome": {
+      "version": "0.5",
+      "schema_url": "https://ngff.openmicroscopy.org/latest/schemas/ome_zarr.schema",
```

Comment:
Is this on purpose? The `schema_url` key is not defined in the specification and I assume a latest URL means the underlying schema will evolve through time.

Reply:
Not on purpose. This needs fixing.

Reply:
Happy to open a PR to remove it unless it's already part of a batch of changes